pkg/costmodel/costmodel.go

```diff
@@ -2193,6 +2214,30 @@ func getStatefulSetsOfPod(pod v1.Pod) []string {
 	return []string{}
 }
 
+func getAllocatableVGPUs(cache clustercache.ClusterCache) (float64, error) {
+	daemonsets := cache.GetAllDaemonSets()
+	vgpuCount := 0.0
+	for _, ds := range daemonsets {
+		dsContainerList := &ds.Spec.Template.Spec.Containers
+		for _, ctnr := range *dsContainerList {
+			if ctnr.Args != nil {
+				for _, arg := range ctnr.Args {
+					if strings.Contains(arg, "--vgpu=") {
+						vgpus, err := strconv.ParseFloat(arg[strings.IndexByte(arg, '=')+1:], 64)
+						if err != nil {
+							return 10.0, fmt.Errorf("cannot parse vgpu allocation string %s: %v", arg, err)
```

Comment:
I think you'd rather continue here. Just because one node has an improperly formatted vgpu setting doesn't mean they all do.

Reply:
Fixed.